public/index.html

```diff
@@ -1,71 +1,76 @@
-    <meta charset="utf-8" />
-    <link rel="icon" href="%PUBLIC_URL%/favicon.svg" />
-    <meta name="viewport" content="width=device-width, initial-scale=1" />
-    <meta property="og:image" content="%PUBLIC_URL%/SocialLogo.jpg" />
+<head>
+    <meta charset="utf-8"/>
+    <link href="%PUBLIC_URL%/favicon.svg" rel="icon"/>
+    <meta content="width=device-width, initial-scale=1" name="viewport"/>
+    <meta content="%PUBLIC_URL%/SocialLogo.jpg" property="og:image"/>
     <meta
-      name="description"
-      content="Crowdsourced info on essential services around you during Covid-19 shutdowns"
+            content="Crowdsourced info on essential services around you during Covid-19 shutdowns"
+            name="description"
     />
 
     <link
-      rel="apple-touch-icon"
-      sizes="180x180"
-      href="%PUBLIC_URL%/favicon/apple-touch-icon.png"
+            href="%PUBLIC_URL%/favicon/apple-touch-icon.png"
+            rel="apple-touch-icon"
+            sizes="180x180"
     />
     <link
-      rel="icon"
-      type="image/png"
-      sizes="32x32"
-      href="%PUBLIC_URL%/favicon/favicon-32x32.png"
+            href="%PUBLIC_URL%/favicon/favicon-32x32.png"
+            rel="icon"
+            sizes="32x32"
+            type="image/png"
     />
     <link
-      rel="icon"
-      type="image/png"
-      sizes="16x16"
-      href="%PUBLIC_URL%/favicon/favicon-16x16.png"
+            href="%PUBLIC_URL%/favicon/favicon-16x16.png"
+            rel="icon"
+            sizes="16x16"
+            type="image/png"
     />
-    <link rel="manifest" href="%PUBLIC_URL%/favicon/site.webmanifest" />
+    <link href="%PUBLIC_URL%/favicon/site.webmanifest" rel="manifest"/>
     <link
-      rel="mask-icon"
-      href="%PUBLIC_URL%/favicon/safari-pinned-tab.svg"
-      color="#f1e932"
+            color="#f1e932"
+            href="%PUBLIC_URL%/favicon/safari-pinned-tab.svg"
+            rel="mask-icon"
     />
-    <link rel="shortcut icon" href="%PUBLIC_URL%/favicon/favicon.ico" />
-    <meta name="msapplication-TileColor" content="#f1e932" />
+    <link href="%PUBLIC_URL%/favicon/favicon.ico" rel="shortcut icon"/>
+    <meta content="#f1e932" name="msapplication-TileColor"/>
     <meta
-      name="msapplication-config"
-      content="%PUBLIC_URL%/favicon/browserconfig.xml"
+            content="%PUBLIC_URL%/favicon/browserconfig.xml"
+            name="msapplication-config"
     />
-    <meta name="theme-color" content="#f1e932" />
+    <meta content="#f1e932" name="theme-color"/>
     <!--
       manifest.json provides metadata used when your web app is installed on a
       user's mobile device or desktop. See https://developers.google.com/web/fundamentals/web-app-manifest/
     -->
-    <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
+    <link href="%PUBLIC_URL%/manifest.json" rel="manifest"/>
+
+    <!-- FontAwesome Pro -->
+    <link crossorigin="anonymous" href="https://pro.fontawesome.com/releases/v5.13.0/css/all.css"
```

Comment:
what is the perf impact of doing this? our existing font awesome usage only imports the icons we use, so that we are not importing the entire icon set. is there any meaningful degradation in perf when we do this?